client.go

```diff
@@ -80,6 +80,153 @@ func NewSync(token, environment, codeVersion, serverHost, serverRoot string) *Cl
 	}
 }
 
+// Log reports an item with the given level. This function recognizes arguments with the following types:
+//
+//	*http.Request
+//	error
+//	string
+//	map[string]interface{}
+//	int
+//	context.Context
+//
+// The string and error types are mutually exclusive.
+// If an error is present then a stack trace is captured. If an int is also present then we skip
+// that number of stack frames. If the map is present it is used as extra custom data in the
+// item. If a string is present without an error, then we log a message without a stack
+// trace. If a request is present we extract as much relevant information from it as we can. If
+// a context is present, it is applied to downstream operations.
+func (c *Client) Log(level string, interfaces ...interface{}) {
```

Comment:
This function was copied as-is from `rollbar.go`, right?

Reply:
we need to have functionality on the client, so they just provide the same functionality, they are all one-liners